sailpoint-identitynow/info.json

```diff
@@ -0,0 +1,803 @@
+            "SLIM",
+            "FULL"
+          ]
+        },
+        {
+          "title": "Limit",
+          "name": "limit",
+          "type": "integer",
+          "tooltip": "Max number of results to return.",
+          "description": "Max number of results to return.",
+          "required": false,
+          "visible": true,
+          "editable": true
+        },
+        {
+          "title": "Offset",
+          "name": "offset",
+          "type": "integer",
+          "tooltip": "Offset into the full result set. Usually specified with limit to paginate through the results.",
+          "description": "Offset into the full result set. Usually specified with limit to paginate through the results.",
+          "required": false,
+          "visible": true,
+          "editable": true
+        }
+      ],
+      "output_schema": [
+        {
+          "id": "",
+          "name": "",
+          "created": "",
+          "modified": "",
+          "sourceId": "",
+          "identityId": "",
+          "attributes": {},
+          "authoritative": "",
+          "description": "",
+          "disabled": "",
+          "locked": "",
+          "nativeIdentity": "",
+          "systemAccount": "",
+          "uncorrelated": "",
+          "uuid": "",
+          "manuallyCorrelated": "",
+          "hasEntitlements": ""
+        }
+      ]
+    },
+    {
+      "operation": "get_account_details",
+      "title": "Get Account Details",
+      "description": "Retrieves a account detail based on the ID.",
+      "category": "investigation",
+      "annotation": "get_account_details",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Account ID",
+          "name": "id",
+          "type": "text",
+          "tooltip": "Specify the ID of the account for which you are retrieving details",
+          "description": "Specify the ID of the account for which you are retrieving details",
+          "required": true,
+          "visible": true,
+          "editable": true
+        }
+      ],
+      "output_schema": {
+        "id": "",
+        "name": "",
+        "created": "",
+        "modified": "",
+        "sourceId": "",
+        "identityId": "",
+        "attributes": {},
+        "authoritative": "",
+        "description": "",
+        "disabled": "",
+        "locked": "",
+        "nativeIdentity": "",
+        "systemAccount": "",
+        "uncorrelated": "",
+        "uuid": "",
+        "manuallyCorrelated": "",
+        "hasEntitlements": ""
+      }
+    },
+    {
+      "operation": "get_account_activities",
+      "title": "Get Account Activities",
+      "description": "This gets a collection of account activities that satisfy the given query parameters.",
+      "category": "investigation",
+      "annotation": "get_account_activities",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Type",
+          "name": "account_type",
+          "type": "text",
```

Comment:
any example of account type?

Reply:
This is an uncertified connector, and API Document only showing required string type.
<img width="428" alt="Screenshot 2022-10-17 at 12 49 26 PM" src="https://user-images.githubusercontent.com/24584941/196113206-e8fe3223-303b-45ef-a0e6-02256ecf6df8.png">